code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -38,81 +39,6 @@
 /mob/dead/new_player/prepare_huds()
 	return
 
-/mob/dead/new_player/Topic(href, href_list[])
```

Comment:
So, TG realised after it was merged, but removing this proc wholesale breaks polls. Please restore lines 41-49, and lines 108-114.